nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -323,6 +324,9 @@ StaticLayer::updateCosts(
   if (!enabled_) {
     return;
   }
+  if (!map_received_) {
+    RCLCPP_ERROR(node_->get_logger(), "Can't update static costmap layer, no map received");
+  }
```

Comment:
Oops, should probably 'return' from the function after printing the ERROR message.